apps/myaccount/src/components/change-password/change-password.tsx

```diff
@@ -315,6 +377,31 @@ export const ChangePassword: FunctionComponent<ChangePasswordProps> = (props: Ch
                             />
                         </Suspense>
                     </Form.Field>
+                    
+                    { passwordValidationConfig.showPasswordValidation  &&
+                        <Form.Field width={ 9 } data-testid={ `${testId}-new-password-validation-field` }>
+                            <div className="password-policy-description">                                
+                                <Icon id = "password-validation-length" className = {getIconProps("password-validation-length").class} color = {getIconProps("password-validation-length").color}/>
```

Comment:
No spaces before/after equal when passing down props. Please use ESLint.

```suggestion
                                <Icon id="password-validation-length" className={getIconProps("password-validation-length").class} color={getIconProps("password-validation-length").color}/>
```